Content.Client/Atmos/Monitor/UI/Widgets/ThresholdControl.xaml.cs

```diff
@@ -115,29 +111,38 @@ public ThresholdControl(string name, AtmosAlarmThreshold threshold, AtmosMonitor
         _enabled.Pressed = !_threshold.Ignore;
     }
 
-    private String LabelForBound(string boundType) //<todo.eoin Replace this with enums
+    private string LabelForBound(string boundType) //<todo.eoin Replace this with enums
     {
         return Loc.GetString($"air-alarm-ui-thresholds-{boundType}");
     }
 
     public void UpdateThresholdData(AtmosAlarmThreshold threshold, float currentAmount)
     {
-        threshold.CheckThreshold(currentAmount, out AtmosAlarmType alarm, out AtmosMonitorThresholdBound which);
+        threshold.CheckThreshold(currentAmount, out var alarm, out var which);
```

Comment:
While our convention is to use `var` instead of explicit types, I think here is a unique exception where leaving the explicit types makes it more clear what this is doing.

Alternatively, rename `which` to `bound`.

Reply:
> While our convention is to use `var` instead of explicit types, I think here is a unique exception where leaving the explicit types makes it more clear what this is doing.
> 
> Alternatively, rename `which` to `bound`.

Good point, will rename it then